resources/wikia/Resources.php

```diff
@@ -89,6 +96,7 @@
 			'wikia.window',
 			'wikia.mw',
 			'wikia.nirvana',
+			'wikia.fbLocale',
```

Comment:
This dependency must also be added here for loader JS. This should eliminate any race condition involved with the loading of this module.